students/1st/feature:1st:kimjihyun/login.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>ZUCKERBUCKS</title>
+    <link rel="stylesheet" href="Style/login.css">
+    <link href="https://fonts.googleapis.com/css2?family=Dancing+Script:wght@500&display=swap" rel="stylesheet">
+</head>
+<body>
+<div class="allContainer">
+    <div class="bucksContainer">
+      <h1 class="title">ZuckerBucks</h1>
+    <div class="inputContainer">
+    <form id="loginForm" method="POST">
+      <input type="text" id="loginID" class="bucksLogin" placeholder="전화번호, 사용자 이름 또는 이메일" required>
+      <input type="password" id="loginPW" class="bucksLogin" placeholder="비밀번호" required>
+      <button class="bucksBtn" disabled='disabled'>로그인</button>
```

Comment:
```suggestion
      <button class="bucksBtn" disabled>로그인</button>
```

이렇게 되어 있어도 됩니다. 위 변경사항은 아래와 같습니다.
```suggestion
      <button class="bucksBtn" disabled=true>로그인</button>
```

Reply:
감사합니다! 장현님 최고